### PR TITLE
Update v3 styles

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

### Top page

- [x]  Build&Earn hero (general)
- [x]  add grid background
- [x]  add dApps list slide arrows
- [x]  add dApps list card hover style
    - [x]  add icon for staking number
    - [x]  add icon for staking amount
        - [x] move svg icon to Logo component
    - [x]  format amount
- [x]  organize data list
- [ ]  animation (P2) - @ayumitk will create another PR for it
- [x]  adjust the leaderboard title position for wide screen
- [x]   adjust padding under the status
- [x]  text animation

### Project page

- [x]  add grid background
- [x]  format total staked balance
- [x]  remove flip animation if no description
- [x]  make icon blur background image smaller
    - [x] adjust icon background position for mobile screen
- [x]  add close button to the fullscreen dApp image (P2)
- [x]  use the same carousel style to the dApp images (P2)

### Owner page

- [x]  add grid background
- [x]  adjust padding under the dApp header 

### Assets page

- [x]  move register now banner to the assets page (P2)
- [x]  modal title change from unbond to unlock

### Common

- [ ]  make titles Italic and ticker
- [ ]  change text from unbond to unlock

---

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

- (ex: Add feature A)

**Fixes**

- (ex: Fix validation function)

**Changes**

- (ex: Change document B)

**To-dos**

> \*Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

**Comment by ayumitk** [comment]
Hi @Kahonnohak, please take a look when you have time.
In this PR, I've changed the dApp card border colour to `gray-2` on the top page for the experiment. I feel the `gray-2` is a bit pale. Should we go maybe `gray-3` or back to `navy-1`? What do you think?

Screenshot 2024-01-24 at 12 52 16 PM

**Comment by Kahonnohak** [comment]
Thank you, yes Navy is probably better isn't it. Just realised we would have to go over other pages and may need to change other places too. Lets not. Thanks for checking

File: src/components/assets/Account.vue [hunk 1]
```diff
@@ -275,6 +276,8 @@ export default defineComponent({
 
     const currentNetworkName = ref<string>(providerEndpoints[currentNetworkIdx.value].displayName);
 
+    const { isDappStakingV3 } = useDappStaking();
```
Comment (impelcrypto) [inline src/components/assets/Account.vue:279]: Let's remove this if you are not using at this file

File: src/components/common/Logo.vue [hunk 2]
```diff
@@ -1,11 +1,36 @@
 <template>
-  <img src="~assets/img/astar_logo_v2.svg" class="icon--logo" />
+  <svg
```
Comment (impelcrypto) [inline src/components/common/Logo.vue:2]: Maybe add this to Astar UI in the future

**Review by impelcrypto — APPROVED** [review]

**Review by bobo-k2 — APPROVED** [review]

File: src/components/assets/Account.vue (hunk 1 above)
Comment (ayumitk) [reply src/components/assets/Account.vue]: @impelcrypto oh, this again. thank you. I've removed it in [e02e1de](https://github.com/AstarNetwork/astar-apps/pull/1150/commits/e02e1def91291bf9ed33cbfa7f2e5b9c65dce0e4)

File: src/components/common/Logo.vue (hunk 2 above)
Comment (ayumitk) [reply src/components/common/Logo.vue:2]: @impelcrypto Agree! Let me add this to my task list after dApp staking v3

**Pull request merged**